extensions/quality/CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
```

Comment:
I think `quality` extension code should be separated from this PR. This comment is valid for all files in `extensions/quality` directory. This will help merging core infrastructure much sooner which will be useful for other PRs like #8 .

Reply:
Done; see https://github.com/linz/STAC/pull/9